test/ufuzz.js

```diff
@@ -573,6 +593,10 @@ function createTypeofExpr() {
         return 'typeof ' + createVarName(MANDATORY, DONT_STORE) + ' != "' + TYPEOF_OUTCOMES[rng(TYPEOF_OUTCOMES.length)] + '"';
       case 4:
         return 'typeof ' + createVarName(MANDATORY, DONT_STORE);
+      case 5:
+      case 6:
+      case 7:
```

Comment:
Only if `default:` does not already exist and probably a dozen other conditions that you or I never would never consider as well.

You know what - it can't work in the general case. You can't typically prove that the condition has a certain number of discrete outcomes.